### DISCUSSION TITLE
How do I input blood/plasma concentration ratio into PK-sim?

### DISCUSSION THREAD

**Discussion by Harris728** [Open-Systems-Pharmacology/Forum#1034] [Questions & Problems] [answered]
Hello guys, we are working on a drug with high blood/plasma concentration ratio. It has nonlinear pharmacokinetics in the low dose range and the t1/2 is long. When we increase the dose high enough, the exposure gradually becomes linear. We think it is because the drug binds to red blood cells tightly and is released slowly. How do I input this property into the model to simulate this case. Besides, what parameters else should I take into consideration.

Thanks.

**Comment by prvmalik** [comment, marked as answer]
You can adjust the Blood:Plasma ratio in the simulation here:

![image](https://user-images.githubusercontent.com/29066045/143982158-7a39acb6-7e02-4ee1-a185-84dc70f88a16.png)

PLEASE NOTE in POPULATION simulations you must instead adjust to the appropriate Partition Coefficient (blood cells/plasma) instead.

Alternatively you could add a protein binding partner in red blood cell intracellular space or red blood cell extracellular membrane

**Reply by Harris728** [reply]
Thank you so much!

**Reply by sajabzughoul** [reply]
@prvmalik 
what if we adjust the blood to plasma ratio in population instead? I DON'T have enough data for Partition Coefficient (blood cells/plasma)

**Reply by prvmalik** [reply]
You use the same value as what is in your individual simulation after applying your manual BP ratio

**Reply by sajabzughoul** [reply]
I have two different blood to plasma ratio values in my simulation; one in hepatic clearance for compound and the second value in the simulation 
based on which value should I go for Partition Coefficient (blood cells/plasma) and other calculation
<img width="960" alt="Untitled" src="https://user-images.githubusercontent.com/45569077/144643112-ee791141-41e2-45ed-a9ac-06dc20ba3792.png">

**Reply by prvmalik** [reply]
The B:P ratio in hepatic clearance section is not used for ANYTHING except calculation of a Kcat parameter for elimination of your compound from the liver. In general I avoid using this parameterization (elect for dummy enzymes instead) because it introduces so much possibility for misinterpretation and inconsistencies for someone trying to review your model.

The true B:P ratio is where I have directed you - this parameter governs partitioning into red blood cells and informs the Partition coefficient (blood cells/plasma). You can see that the partition coefficient changes when you change the B:P ratio in the distribution > partition coefficient section but NOT when you change the B:P ratio in your hepatic clearance section.
